src/cronjobs/monthlyActivity.tsx

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { SubscriberRow } from "knex/types/tables";
+import {
+  getSubscribersWaitingForMonthlyEmail,
+  updateMonthlyEmailTimestamp,
+} from "../db/tables/subscribers";
+import { initEmail, sendEmail } from "../utils/email";
+import { renderEmail } from "../emails/renderEmail";
+import { MonthlyActivityEmail } from "../emails/templates/monthlyActivity/MonthlyActivityEmail";
+import { getEmailL10n } from "../emails/getEmailL10n.node";
+import { sanitizeSubscriberRow } from "../app/functions/server/sanitize";
+import { getDashboardSummary } from "../app/functions/server/dashboard";
+import { getLatestOnerepScanResults } from "../db/tables/onerep_scans";
+import { getSubscriberBreaches } from "../app/functions/server/getSubscriberBreaches";
+import { getLocale } from "../app/functions/universal/getLocale";
+
+void run();
+
+async function run() {
+  const batchSizeEnvVar = Number.parseInt(
+    process.env.MONTHLY_ACTIVITY_EMAIL_BATCH_SIZE ?? "",
+    10,
+  );
+  const batchSize = Number.isNaN(batchSizeEnvVar) ? 10 : batchSizeEnvVar;
```

Comment:
I figured I'd start with (configurable) small batches so that we can just run the cron job more often and not overload the server with a single big one, since we're doing multiple queries per user. Since this is limited to Plus users for now, there's not that many accounts we need to get through anyway. Also makes it easier to recover from errors.